src/utils/status/host/hostStatus.js

```diff
@@ -1,6 +1,5 @@
-import { HOST_STATUS_READY } from './constants';
+import { getHostStatus as getHostStatusSelector } from '../../../selectors';
 
-// TODO(honza): Fill this out
 export const getHostStatus = host => ({
-  status: HOST_STATUS_READY,
+  status: getHostStatusSelector(host),
```

Comment:
this is going the other way around. It should be implemented here.

Reply:
I'm not sure I follow. Could you explain a bit more? 

There's really nothing to implement. It just passes the value into this status object.

Reply:
sorry, I wrote it confusingly. I just meant that https://github.com/kubevirt/web-ui-components/pull/293 needs to be resolved first and it will call the status selector.